### DISCUSSION TITLE
Airflow DAG not getting triggered.

### DISCUSSION THREAD

**Discussion by saurasingh** [apache/airflow#42379] [Q&A]
**Airflow version**: 2.7.1

We have created one airflow dag to do some schema cleanup in Snowflake. It's on schedule to run once a week. When we run this DAG in our development environment it works fine and gets triggered by itself, no matter how frequently we schedule it. But once we deploy this in our STG and PRD environment it never gets triggered. If we manually run it works. 
All our environment are on the same version of Airflow

**Comment by andresitorusz** [comment]
Could you show the DAG configuration? I mean the DAG  code snippet?

But, I assume that you are using dynamic `start_date`.
